.github/ISSUE_TEMPLATE.md

```diff
@@ -25,13 +37,16 @@ PASTE TRACEBACK HERE
 
 ```
 
-## Package Versions
+## Versions
+<!--- You can get this information from Help > About Spyder...
+or (if Spyder won't launch) the "conda list" command
+from the Anaconda Prompt/Terminal/command line. --->
 
-* Spyder:
-* Python:
-* Qt:
-* PyQt:
-* Operating System:
+* Spyder version:
+* Python version:
+* Qt version:
+* PyQt version:
+* OS name and version:
```

Comment:
`OS` is a term some people could not understand. So please revert this to `Operating System:`

Reply:
The problem (and I think I mentioned this before as well) is that we're out of characters to play around with, as aside from a small buffer I built in for one or two lines more of deps, the report starts getting cut off or outright producing errors when trying to send it to github, and I already cut and cut and cut the issue checklist down to under 1/3 of its original length as well as the disclaimer, headers etc . The real issue is that the dependency list takes up ~3-4x the characters it should due to all the fancy symbols used needed percent encoding; even a small change here (eliminating one or two deps that rarely matter; using one or two unnecessary symbols in the plain text version, etc. will cut dozens or even hundreds of characters from the final URL.

Please advise what you'd like me to do—fix the plain text display of the deps? Convert this to use the new error dialog? Just risk it and add back the chars? If I do the latter, adding even one more dep (as is discussed elsewhere) will likely break the whole URL based error reporting mechanism, or at least come dangerously close.

Reply:
Let me know what you'd like me to do about this, @ccordoba12 . Thanks!

Reply:
Sorry for the delay in coming back to you. Please remove this entry in the new list:

> Completed the problem description, steps to reproduce, ...

That's kind of obvious and it should be enough to add the characters that we need.

Reply:
Sorry for the confusion; I meant to make the above initial comment on the mainwindow review item instead. That entry only exists (in that lengthy form at least) in the issue report template; the relevant text for the Help menu Report Issue item is in ``mainwindow``. However, 

> Completed all applicable sections below

while much shorter, does exist and, once percent encoded, is indeed around the same length as a dependency line, much longer than "versions". Unfortunately it isn't as obvious as one would think as many reporters don't do it, but at least it is something we can easily see for ourselves. I'll go ahead and eliminate it and spell out versions and OS, and push the changes momentarily; that should at least give us a little breathing room for the time being.